libs/interpret/src/lib/MLIDashboard/utils/buildColumnRanges.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  DatasetCohortColumns,
+  IJointMeta,
+  JointDataset
+} from "@responsible-ai/core-ui";
+import { IColumnRange, RangeTypes } from "@responsible-ai/mlchartlib";
+import _ from "lodash";
+
+// a workaround for explanation dashboard to use the new filter in CohortEditor
+export function buildColumnRanges(jointDataset: JointDataset): {
+  [key: string]: IColumnRange;
+} {
+  const range: {
+    [key: string]: IColumnRange;
+  } = {};
+  for (const [key, value] of Object.entries(jointDataset.metaDict)) {
+    const columnRange = getRange(key, value, jointDataset.dataDict);
+    if (columnRange) {
+      if (key === JointDataset.RegressionError) {
```

Comment:
you could simplify here by creating a dictionary with JointDataset keys and DatasetCohortColumns values - then index into that to get the value to set the columnRange

Reply:
sounds good. will update in a separate PR, thank you!